### ISSUE TITLE
CKV_DOCKER_11 false positive when `--platform` is used

### ISSUE THREAD

**Issue by LinguineCode** [bridgecrewio/checkov#5161]
**Describe the issue**

CKV_DOCKER_11 false positive when `--platform` is used (possibly other arguments as well)

For reference: _"CKV_DOCKER_11: "Ensure From Alias are unique for multistage builds."_  In other words, make sure you add `as myAlias` at the end of your `FROM` line

**Examples**

This will PASS as expected:
`FROM node:16 as build`

Now, add `--platform` and it will FAIL:
`FROM --platform=linux/amd64 node:16 as build`

**Version (please complete the following information):**
```
> checkov -v      
2.3.240
```

**Additional context**

**Comment by LinguineCode** [comment]
> hey @LinguineCode thanks for reaching out.
> 
> I tried to reproduce it, but the mentioned check is passing for me. Can you please provide your `checkov` version and if possible a more complete example?

My pleasure, here you go:

> ```
> > checkov -v
> 2.3.240
> ```

> ```
> > checkov --quiet -f Dockerfile
> dockerfile scan results:
> 
> Passed checks: 6, Failed checks: 1, Skipped checks: 0
> 
> Check: CKV_DOCKER_11: "Ensure From Alias are unique for multistage builds."
> 	FAILED for resource: Dockerfile.FROM
> 	File: Dockerfile:1-1
> 	Guide: https://docs.bridgecrew.io/docs/ensure-docker-from-alias-is-unique-for-multistage-builds
> 
> 		1 | FROM --platform=linux/amd64 node:16 as FOO
>
> >
> ```

Here's the `Dockerfile`:
```Dockerfile
FROM --platform=linux/amd64 node:16 as FOO
RUN npm install

FROM --platform=linux/amd64 node:16 as BAR
RUN npm run

USER nobody
HEALTHCHECK CMD curl --fail http://localhost:3000 || exit 1
```

**Comment by gruebel** [comment]
thanks, that did the trick 😄
